.github/workflows/go_generate.yml

```diff
@@ -28,7 +28,14 @@ jobs:
         run: |
           mkdir gentools
           go build -o gentools/stringer golang.org/x/tools/cmd/stringer
```

Comment:
I don't think we need this anymore. Awhile back I updated all the raw "stringer" calls to use "go run golang.org/x/tools/cmd/stringer" instead.

Reply:
Ack. Will fix in separate commit in this PR.